raaghu-elements/src/rds-checkbox/rds-checkbox.tsx

```
@@ -46,147 +46,144 @@ export interface RdsCheckboxProps {
 
 const RdsCheckbox: React.FC<RdsCheckboxProps> = (props) => {
   const [check, setCheck] = useState(props.checked);
+  const [isHovered, setIsHovered] = useState(false);
```

ESLint

[error] Expected indentation of 4 spaces but found 2.

(indent)